src/Cache/MongoStore.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace MongoDB\Laravel\Cache;
+
+use Illuminate\Cache\DatabaseStore;
+use MongoDB\Laravel\Collection;
+use MongoDB\Laravel\Connection;
+
+use function assert;
+
+class MongoStore extends DatabaseStore
+{
+    public function __construct(
+        Connection $connection,
+        string $table,
+        string $prefix = '',
+        string $lockTable = 'cache_locks',
+        array $lockLottery = [2, 100],
+        int $defaultLockTimeoutInSeconds = 86400,
+    ) {
+        parent::__construct($connection, $table, $prefix, $lockTable, $lockLottery, $defaultLockTimeoutInSeconds);
+    }
+
+    public function put($key, $value, $seconds): bool
+    {
+        $key = $this->prefix . $key;
+        $value = $this->serialize($value);
+        $expiration = $this->getTime() + $seconds;
+        $collection = $this->table()->raw(null);
+        assert($collection instanceof Collection);
+
+        $result = $collection->updateOne(
+            ['key' => ['$eq' => $key]],
```

Comment:
Why the explicit use of `$eq` instead of the shorter variant?
```suggestion
            ['key' => $key],
```

Reply:
I assumed that the verbose version would be more explicit to avoid edge cases.